snuba/web/views.py

```diff
@@ -298,16 +299,33 @@ def config(fmt: str = "html") -> Union[Response, RespTuple]:
             )
         else:
             assert http_request.method == "POST"
-            state.set_configs(
-                json.loads(http_request.data),
-                user=http_request.headers.get("x-forwarded-email"),
-            )
-            return (
-                json.dumps(state.get_raw_configs()),
-                200,
-                {"Content-Type": "application/json"},
-            )
-
+            try:
+                state.set_configs(
+                    json.loads(http_request.data),
+                    user=http_request.headers.get("x-forwarded-email"),
+                )
+                return (
+                    json.dumps(state.get_raw_configs()),
+                    200,
+                    {"Content-Type": "application/json"},
+                )
+            except MismatchedTypeException as exc:
+                return (
+                    json.dumps(
+                        {
+                            "error": {
+                                "type": "client_error",
+                                "message": "Existing value and New value have different types. Use option force to override check",
```

Comment:
I don't see a way to force it using the web API.

```suggestion
                                "message": "Existing value and New value have different types.",
```

Reply:
Good catch, will add it. That endpoint isn't used anywhere I believe, just something we can move over to admin.

Reply:
Hmmm, this doesn't fit easily without changing the format of this API. I'm going to omit this since it's getting deprecated and not being used.